app/Support/Rules/ContainsRegularCharacter.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Support\Rules;
+
+use Closure;
+use Illuminate\Contracts\Validation\ValidationRule;
+
+class ContainsRegularCharacter implements ValidationRule
+{
+    public function validate(string $attribute, mixed $value, Closure $fail): void
+    {
+        // Check for at least one letter, number, punctuation mark, or symbol.
+        $requireOneRegularCharacter = '/[\p{L}\p{N}\p{P}\p{S}]/u';
```

Comment:
Still able to submit non-visible characters as HTML escape codes: `&#x200B;`

Reply:
Try hex C2AD (search message_threads for id = 159237 and copy the `title` value).

![Screenshot 2024-09-07 at 6 14 04 PM](https://github.com/user-attachments/assets/9060b949-f3fe-483a-be1a-f7819b098e62)

Reply:
Added HTML entity decoding and another test case in latest to cover this.